python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -1805,11 +1826,32 @@ async def set_atspectrograph_corrections(self, azimuth, elevation):
 
         Can only be applied if hexapod correction is also applied
         """
-        self.log.debug("At the beginning of set_atspectrograph_corrections")
+        self.log.debug(
+            f"At the beginning of set_atspectrograph_corrections with "
+            f"{self.atspectrograph_corrections_required} correction "
+            f"loops required. "
+        )
 
-        if self.can_move():
-            # publish new detailed state
-            self.log.debug("Applying Spectrograph Corrections, if required")
+        if (
+            self.atspectrograph_corrections_required == 0
+            and self.focus_offset_yet_to_be_applied <= 1e-12
+        ):
+            self.log.warning(
+                "No atspectrograph corrections loaded but focus_offsets are non-zero"
+            )
+        if (
+            self.atspectrograph_corrections_required == 0
+            and np.max(np.abs(self.pointing_offsets_yet_to_be_applied)) <= 1e-12
+        ):
+            self.log.warning(
+                "No atspectrograph corrections loaded but focus_offsets are non-zero"
```

Comment:
Same comment about offset not being zero but comparison is with really small number.

Reply:
Same as above, except i also changed  `focus_offsets` to `pointing_offsets`